templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -288,42 +382,16 @@ contents:
       done
 
       echo "ERROR: Failed to activate ovs-if-br-ex NM connection"
-      # if we made it here networking isnt coming up, revert for debugging
-      set +e
-      nmcli conn down ovs-if-br-ex
-      nmcli conn down ovs-if-phys0
-      nmcli conn up $old_conn
       exit 1
     elif [ "$1" == "OpenShiftSDN" ]; then
       # Revert changes made by /usr/local/bin/configure-ovs.sh.
-      # Remove OVS bridge "br-ex". Use the default NIC for cluster network.
-      iface=""
-      if nmcli connection show ovs-port-phys0 &> /dev/null; then
-        iface=$(nmcli --get-values connection.interface-name connection show ovs-port-phys0)
-        nmcli c del ovs-port-phys0 
-      fi
-
-      if nmcli connection show ovs-if-phys0 &> /dev/null; then
-        nmcli c del ovs-if-phys0
-      fi
+      rm_nm_conn_files
```

Comment:
Shall we also backport the change we made in #2704?

Reply:
I would guess so, but should we do it through the same BZ/PR?

The 4.8 backport is already well on it's way: https://github.com/openshift/machine-config-operator/pull/2644

Reply:
#2644 is still not merged yet. Without #2704, we may introduce regression to 4.7 and 4.8. So I prefer we put them all together.

Reply:
@rbbratta what do you think?

Don't we need separate backport BZs administratively speaking?